docs/configuration.asciidoc

```diff
@@ -633,6 +633,26 @@ This option supports the wildcard `*`, which matches zero or more characters. Ex
 | `ELASTIC_APM_DISABLE_METRICS` | `ElasticApm:DisableMetrics`
 |============
 
+[float]
+[[config-cloud-provider]]
+==== `CloudProvider` (added[1.7.0])
+
+Allows you to specify which cloud provider should be assumed for metadata collection. By default, the agent will attempt to detect the cloud provider and, if that fails, use trial and error to collect the metadata.
+
+Valid options are `"aws"`, `"gcp"`, and `"azure"`. If this config value is set to `"false"`, no cloud metadata will be collected.
```

Comment:
I see we have this sentence also in the python agent. To me it's a bit contradicting that it lists the 3 cloud provider names as "valid option", and then in the next sentence it says what happens if it's `false`.

But maybe this is a problem with me :)

Reply:
Agreed that this wording is a bit funky. Maybe something like:

```suggestion
Valid options are `"aws"`, `"gcp"`, `"azure"`, and `"false"`. If this config value is `null` or set to `"false"`, no cloud metadata will be collected.
```